src/main/scripts/cd/deploy.sh

```diff
@@ -4,3 +4,5 @@ echo       Running $0
 echo *-*-*-*-*-*-*-*-*-*-*-*-*-*
 
 mvn -P release deploy -DskipTests=true -B -V -s travis-settings.xml
+apt update && apt install -y python3-pip && pip3 install -r requirements.txt
```

Comment:
the file requirements is not here, should be added through the entrypoint script.

Reply:
Fixed